lib/src/main/java/io/ably/lib/realtime/ChannelBase.java

```diff
@@ -624,7 +663,7 @@ public synchronized void setSuspended(ErrorInfo reason, boolean notifyStateChang
             Log.v(TAG, "setSuspended(); channel = " + name);
             presence.setSuspended(reason);
             setState(ChannelState.suspended, reason, false, notifyStateChange);
-            failQueuedMessages(reason);
+            // failQueuedMessages(reason);
```

Comment:
why is this removed?

Reply:
Sorry - this was a result of me furiously commenting out and trying different things and forgetting when cleaning up the code. I believe I commented this out when the channels would go into suspended state as attach wouldn't happen. I put it back f525dcac9fcaef94d1b7893860adb125d9eb307f